include/bluetooth/mesh/light_ctrl.h

```diff
@@ -14,6 +14,7 @@
 #define BT_MESH_LIGHT_CTRL_H__
 
 #include <bluetooth/mesh/properties.h>
+#include <bluetooth/mesh/sensor.h>
```

Comment:
the include of this file probably need to be replaced by include of sensor_types.h. The define in sensor.h will probably be duplicate of what will be in sensor_types.h.